src/test/java/aquality/selenium/waitings/ConditionalWaitTests.java

```diff
@@ -144,23 +153,27 @@ public void testExceptionShouldBeCatchedConditionIsMetAndDefaultTimeoutIsNotOver
                     }, waitForTimeoutCondition, waitForTimeoutPolling,
                     "Conditional should be true", Collections.singleton(IllegalArgumentException.class));
         } catch (org.openqa.selenium.TimeoutException e){
-            double duration = timer.stop();
+            DurationSample durationSample = new DurationSample(timer.duration(), waitForTimeoutCondition, 7);
+            BrowserManager.getBrowser().quit();
 
-            assertTrue(duration > waitForTimeoutCondition && duration < 2 * waitForTimeoutCondition);
+            assertTrue(durationSample.isDurationBetweenLimits(), durationSample.toString());
         }
     }
 
     @Test
     public void testTimeoutExceptionShouldNotBeThrownIfDriverConditionIsMetAndTimeoutIsNotOver(){
+        Timer timer = new Timer();
+
         ConditionalWait.waitFor((driver) ->
                 {
                     timer.start();
                     return true;
                 }, waitForTimeoutCondition, waitForTimeoutPolling,
                 "Conditional should be true", Collections.singleton(IllegalArgumentException.class));
-        double duration = timer.stop();
+        DurationSample durationSample = new DurationSample(timer.duration(), waitForTimeoutCondition, 0);
```

Comment:
0 for deviation is set 4 times in this class, should it be a private method or a default value for this parameter?